### PR TITLE
[Docs] Add Santa's Helper tuples story

### PR DESCRIPTION
**Pull request by wneumann**
Adding Santa's Helper tuples story used in the Swift track

### CODE REVIEW

File: reference/stories/tuples.santas-helper.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+The story facilitates the exploration of [tuples][type-tuple] in a language. It does this by requiring the student to extract values from tuples, create tuples, and work with named tuple fields over the course of two tasks.
+
+- Convert coordinates to polar
+- Merge two database records
+
+## Implementations
+
+- [Swift: tuples][implementation-swift] (reference implementation)
+
```
Comment (ErikSchierboom) [inline reference/stories/tuples.santas-helper.md:17]:
```suggestion

## Related

- [`types/tuple`][type-tuple]

```

**Review by ErikSchierboom — APPROVED** [review]
Thanks!

File: reference/types/tuple.md [hunk 2]
```diff
@@ -14,11 +14,17 @@ This exercise asks student to create and maintain few functions which use tuples
...
+### Santa's Helper
+
+This exercise asks the student to the student to implement a pair of functions which extract values from tuples, create tuples, and work with named tuple fields.
+
 #### Implementations
 
 | Track  | Exercise                        | Changes |
 | ------ | ------------------------------- | ------- |
 | Elixir | [tuples][implementation-elixir] | None    |
+| Swift  | [tuples][implementation-swift]  | None    |
```
Comment (angelikatyborska) [inline reference/types/tuple.md:26]: Shouldn't there be two separate tables, each for each story?

Comment (wneumann) [reply reference/types/tuple.md:26]: I do believe you are correct. It certainly makes more sense that way.

I'll post a correction; thanks for catching this.

**Pull request merged**